guides/common/modules/con_supported-client-architectures-configuration-management.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Supported-Client-Architectures-Configuration-Management_{context}"]
+= Supported Client Architectures: Configuration Management
+
+Supported combinations of major versions of {RHEL} and hardware architectures for configuration management with {Project}.
```

Comment:
We should write in complete sentences. Please reword the intros to all these "Supported" modules into complete sentences. This will help users understand them better and it also will be easier to localize.

https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

Reply:
Addressed with https://github.com/theforeman/foreman-documentation/pull/2607/commits/a07a25b33556ce35e1331d070a6386e12e5d7ce0